### ISSUE TITLE
Question (not an issue): Using snorkel.classification

### ISSUE THREAD

**Issue by haroonhassan** [snorkel-team/snorkel#1528]
Hi, was wondering if there are any examples of snorkel classifier out of the box? The tutorials showcase the keras api with probabilistic labels, but curious to compare with the snorkel package. Any help appreciated. Thanks.

**Comment by bhancock8** [comment]
Yep! The [multitask tutorial](https://github.com/snorkel-team/snorkel-tutorials/blob/master/multitask/multitask_tutorial.ipynb) demonstrates how to use the `MultitaskClassifier` that comes with Snorkel. 

I should note though that our general recommendation is to use other great third-party tools (such as Keras, scikit-learn, Ludwig, etc.) for single-task problems. The `MultitaskClassifier` is designed specifically for tackling advanced use cases (such as multi-task learning, multi-modal inputs, or slice-aware models), so while you can certainly use it for a simple single-task use case where its advanced features aren't necessarily required, you may find it a little more cumbersome to use than models that are targeted specifically at single tasks.
